schema/gitlab.schema.graphqls

```diff
@@ -172,6 +172,13 @@ enum PipelineStatusEnum {
   WAITING_FOR_RESOURCE
 }
 
+enum ApprovalRuleType {
```

Comment:
mind adding a link to https://docs.gitlab.com/ee/api/graphql/reference/#approvalruletype and adding a comment to each enum value from those docs?